### PR TITLE
Parcel page

### PR DESCRIPTION
**Pull request by an-uhryn**

### CODE REVIEW

File: src/api/thegraph.js [hunk 1]
```diff
@@ -2,8 +2,21 @@ import { ApolloClient, InMemoryCache, HttpLink } from '@apollo/client';
...
+    userQuery,
+    realmQuery,
+    auctionQuery,
+    raffleQuery,
+    raffleEntrantsQuery,
+    raffleWinsQuery,
+    listedParcelsQuery,
+    parselQuery,
+    clientParselQuery,
+    listedParcelQuery, getParcelHistoricalPricesQuery
```
Comment (andkoval) [inline src/api/thegraph.js:18]: last one :)

**Comment by andkoval** [comment]
@an-uhryn let's use single-quotes everywhere

**Review by andkoval — CHANGES_REQUESTED** [review]

File: src/api/thegraph.js (hunk 1 above)
Comment (an-uhryn) [reply src/api/thegraph.js]: Done))

**Comment by an-uhryn** [comment]
> @an-uhryn let's use single-quotes everywhere

Lets do it)

**Review by andkoval — APPROVED** [review]

**Pull request merged**